IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib

```diff
@@ -54,21 +54,32 @@
                             <color key="textColor" red="0.60392156860000001" green="0.60392156860000001" blue="0.60392156860000001" alpha="1" colorSpace="calibratedRGB"/>
                             <nil key="highlightedColor"/>
                         </label>
+                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="e4A-OW-CdX">
+                            <rect key="frame" x="300" y="8" width="64" height="32"/>
```

Comment:
Can we make the button slightly smaller and align better with our design spec? 

Perhaps something in this direction: https://www.figma.com/file/IcXFkmJ8atZKdBAFX5a6xEuG/Design-System?type=design&node-id=1968%3A86&mode=design&t=fgPSKnWk1NjEEPMu-1
<img width="411" alt="Screenshot 2023-10-26 at 16 15 52" src="https://github.com/avohq/ios-analytics-debugger/assets/3513968/032eaba7-688e-4c3d-90f6-c300ad3b2668">

Reply:
I'd make it slightly smaller, and the text semibold 💅